README.md

```diff
@@ -15,43 +20,50 @@ of Zcash transactions; depending on the speed of your computer and network
 connection, the synchronization process could take a day or more once the
 blockchain has reached a significant size.
 
-Security Warnings
------------------
+<p align="center">
+  <img src="doc/imgs/zcashd_screen.gif" height="500">  
+</p>
+
+#### :lock: Security Warnings
 
 See important security warnings on the
 [Security Information page](https://z.cash/support/security/).
 
 **Zcash is experimental and a work-in-progress.** Use at your own risk.
 
-Deprecation Policy
-------------------
+####  :ledger: Deprecation Policy
 
-This release is considered deprecated 16 weeks after the release day. There
+> This release is considered deprecated 16 weeks after the release day. There
```

Comment:
Since 1.1.2, we've removed the `-disabledeprecation` feature so 

> and can be explicitly disabled.

 is no longer true

Reply:
Fixed